index.html

```diff
@@ -1,203 +1,190 @@
-                                <span>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Autem porro nesciunt,
-                                    nam
-                                    corporis culpa nihil vero modi est quod distinctio, expedita blanditi!</span>
-                            </div>
-
-                        </div>
-                        <div class="hiddenPort">
-                            <div class="SpeakerPort"><ion-icon id="picdec" name="film"></ion-icon>
-                                <img src="Images/istockphoto-502581380-612x612.jpg" alt="Black American Speaker">
-                                <div class="port"> <span> <b>Anita Ama</b> <br> <span class="portTitle">Lorem ipsum
-                                            dolor
-                                            sit
-                                            amet consectetur adipisicing elit. Dolor</span> <br><span
-                                            id="portdash">____</span><br><br></span>
-                                    <span>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Autem porro
-                                        nesciunt,
-                                        nam
-                                        corporis culpa nihil vero modi est quod distinctio, expedita blanditi!</span>
-                                </div>
-
-                            </div>
-
-                            <div class="SpeakerPort"><ion-icon id="picdec" name="film"></ion-icon>
-                                <img src="Images/istockphoto-615279718-612x612.jpg" alt="Black American Speaker">
-                                <div class="port"> <span> <b>Anita Ama</b> <br> <span class="portTitle">Lorem ipsum
-                                            dolor
-                                            sit
-                                            amet consectetur adipisicing elit. Dolor</span> <br><span
-                                            id="portdash">____</span><br><br></span>
-                                    <span>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Autem porro
-                                        nesciunt,
-                                        nam
-                                        corporis culpa nihil vero modi est quod distinctio, expedita blanditi!</span>
-                                </div>
-
-                            </div>
+            <button id="register" class="Join">
+              <a class="butt" href="#">Click Here To Register!</a>
+            </button>
+          </div>
+        </section>
 
-                        </div>
+        <section>
+          <div id="featuredSpeakers" class="featuredSpeakers">
+            <h1>
+              Featured Speakers<ion-icon
+                id="featicon"
+                name="chevron-down-outline"
+              ></ion-icon>
+            </h1>
+            <div class="featureContainer">
+            </div>

```

Comment:
-  [ ] As I mentioned above :point_up_2: please stick to the Behance design but there are some spacing and font-size problems so kindly fix these issues as well :+1: check below :point_down:  the comparison between your design and Behance design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-07-07 15-48-55](https://github.com/Mike47ip/JavascriptCapstone/assets/45226848/40965aac-a833-4f9b-8416-f5ed204d8ab0)

**Behance Design**
![Screenshot from 2023-07-07 15-48-32](https://github.com/Mike47ip/JavascriptCapstone/assets/45226848/57f38590-fd78-4f5b-869b-ea6dbf09cb4e)